tests/test_models.py

```diff
@@ -905,6 +905,58 @@ def test_sync_to_db_default_view(self, mock_now):
         self.assertIsNotNone(orm_dag.default_view)
         self.assertEqual(orm_dag.get_default_view(), "graph")
 
+    def test_upstream_deps_successful(self):
+        dag1 = DAG(dag_id='dag1', start_date=DEFAULT_DATE)
+        dag2 = DAG(dag_id='dag2', start_date=DEFAULT_DATE)
+        dag3 = DAG(dag_id='dag3', start_date=DEFAULT_DATE)
+        dag4 = DAG(dag_id='dag4', start_date=DEFAULT_DATE)
+        dag5 = DAG(dag_id='dag5', start_date=DEFAULT_DATE)
```

Comment:
I think wanted to add an extra DAG to test what happens when `state=State.RUNNING`?

Reply:
@jimmyatwork added extra DAG to test for `RUNNING` state.